spec/factories/users.rb

```diff
@@ -2,16 +2,21 @@
 
 FactoryGirl.define do
   factory :user do
-    id 1
+    sequence(:id, 5)
     name "Diego Martinez"
     email "diego@email.com"
     password "password"
 
     factory :another_user do
-      id 2
       name "Heitor Reis"
       email "hr@email.com"
       password "password"
     end
+    
+    factory :mezuro_user do
+      name "Mezuro Default user"
+      email "mezuro@librelist.com"
+      password Devise.friendly_token.first(10)
```

Comment:
We need to document that the user should set a new password from Rails' console.